pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "vmlinux.h"
+#include "bpf_helpers.h"
+
+struct ct_key {
+    __u32 src_ip;
+    __u32 dst_ip;
+    __u16 src_port;
+    __u16 dst_port;
+    __u8 protocol;
+};
+
+struct ct_value {
+    __u64 timestamp;
+    __u32 flags;
+    __u8 isClosed;
+};
+
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
```

Comment:
I am not sure hashmap would be the ideal solution here

Reply:
`BPF_MAP_TYPE_LRU_HASH` we should ideally be using this, and have a mechanism inside to detect if an entry is expired.